file_entity.routing.yml

```diff
@@ -144,3 +144,14 @@ file_entity.settings:
     _title: 'File settings'
   requirements:
     _permission: 'administer files'
+
+file_entity.file_add_archive_form:
+  path: /file/add/upload/archive
+  defaults:
+    _form: '\Drupal\file_entity\Form\FileAddArchiveForm'
+    _title: 'Add files'
```

Comment:
I'd rather call this differently. "Upload files archive", "Upload archive" or something in that direction.